hokusai/production.yml

```diff
@@ -103,3 +103,75 @@ spec:
                     operator: In
                     values:
                     - background
+
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: opstools-cleanup-pods-by-name
+spec:
+  schedule: "0 * * * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      backoffLimit: 1
+      template:
+        metadata:
+          annotations:
+            "cluster-autoscaler.kubernetes.io/safe-to-evict": "false"
+        spec:
+          containers:
+            - name: opstools-cleanup-pods-by-name
+              image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/opstools:production
+              command: ["src/kubernetes_cleanup_pods_by_name/cleanup.py", "hokusai-run", "6", "--force"]
```

Comment:
Suggested minor improvement: filtering to `-hokusai-run-` would be slightly more conservative.

Reply:
Thanks, done in https://github.com/artsy/opstools/pull/29/commits/d0a978b9d93d150c5de37eb57ea83cd7030829c8#diff-8ba030368b0c3dfd60ec6bedbbb859e1ca856d64344e6e72b59ffeac4d310d3fR126